### PR TITLE
update code to support elazarl/go-bindata-assetfs's AssetInfo

### PR DESCRIPTION
**Pull request by fredipevcin**

### CODE REVIEW

**Comment by mephux** [comment]
@fredipevcin - looks good to me, thanks.  /cc @Unknwon

File: bindata.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ package bindata
 import (
 	"github.com/elazarl/go-bindata-assetfs"
 	"gopkg.in/macaron.v1"
+	"os"
```
Comment (Unknwon) [inline bindata.go]: Please move this line up like:

```go
import (
  	"os"

  	"github.com/elazarl/go-bindata-assetfs"
  	"gopkg.in/macaron.v1"
)
```

Comment (fredipevcin) [inline bindata.go]: Hi, I've fixed it and also ran `gofmt`.

**Comment by Unknwon** [comment]
Thank you!

**Pull request merged**